src/components/common/BackLink.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Link from '@material-ui/core/Link';
+import { NavLink } from 'react-router-dom';
+
+const BackLink = ({ to }: { to: string }) => (
+  <Link color="secondary" variant="body2" component={NavLink} to={to}>
+    &#60; Back
```

Comment:
just wondering what the `&#60;` is doing, and if we need it?

Reply:
it's the html entity for the less than character e.g. `<` - it's included in the mocks so the back link/button looks like this
![image](https://user-images.githubusercontent.com/59516223/91589906-7ccd1480-e928-11ea-82be-d3d54911be71.png)